Pipeline/PackageBuildOutputs/README.md

```diff
@@ -146,8 +152,108 @@ PackageBuildOutputs console output
+*** Number of build outputs to package: 1
+** Copying build outputs to temporary package directory /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/tempPackageDir
+   Copy ADO.RETIREME.MAIN.BLD.LOAD(EBUD01) to /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/tempPackageDir/ADO.RETIREME.MAIN.BLD.LOAD/EBUD01.LOAD with DBB Copymode LOAD
+** Generate Wazi Deploy Application Manifest file to /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/tempPackageDir/waziDeployManifest.yaml
+---
+apiVersion: "wazideploy.ibm.com/v1"
+kind: "ManifestState"
+metadata:
+  name: "retirementCalculator"
+  description: "retirementCalculator"
+  version: "rel-2.0.0"
+  annotations:
+    creationTimestamp: "20240304.015055.050"
+    scmInfo:
+      type: "git"
+      uri: "git@ssh.dev.azure.com:v3/IBM-DAT/retirementCalculator/retirementCalculator"
+      branch: "main"
+      shortCommit: "cac60e95685575fab15a583d2a3966a087b58b38"
+    packageInfo: null
+artifacts:
+- name: "EBUD01"
+  description: "retirementCalculator/cobol/EBUD01.cbl"
+  properties:
+  - key: "githash"
+    value: "cac60e95685575fab15a583d2a3966a087b58b38"
+  - key: "giturl"
+    value: "git@ssh.dev.azure.com:v3/IBM-DAT/retirementCalculator/retirementCalculator"
+  type: "LOAD"
+  hash: "cac60e95685575fab15a583d2a3966a087b58b38"
+
+** Generate package build report order file to /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/tempPackageDir/buildReportOrder.txt
+** Copy packaging properties config file to /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/tempPackageDir/packageBuildOutputs.properties
+** Creating tar file at /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/retirementCalculator.tar
+   Executing [sh, -c, tar cUXf /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/retirementCalculator.tar *]
+** Package successfully created at /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/retirementCalculator.tar
+** Adding files with file pattern *.log from /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs to /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/retirementCalculator.tar
+   Executing [sh, -c, tar rUXf /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/retirementCalculator.tar *.log]
+** List package contents.
+   Executing [sh, -c, tar tvf /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/retirementCalculator.tar]
+drwxr-xr-x   1 BPXROOT  TIVUSR         0 Mar  4 13:50 ADO.RETIREME.MAIN.BLD.LOAD/
+-rwxr-xr-x   1 BPXROOT  TIVUSR     49152 Mar  4 13:50 ADO.RETIREME.MAIN.BLD.LOAD/EBUD01.LOAD
+-rw-r--r--   1 ADO      TIVUSR      4239 Feb 15 10:19 BuildReport.json
+-rw-r--r--   1 BPXROOT  TIVUSR        97 Mar  4 13:50 buildReportOrder.txt
+-rw-r--r--   1 BPXROOT  TIVUSR      1515 Mar  1 17:54 packageBuildOutputs.properties
+-rw-r--r--   1 BPXROOT  TIVUSR       790 Mar  4 13:50 waziDeployManifest.yaml
```

Comment:
The file name for WD 3.0.2 must be at the root level of the tar file and named `wazideploy_manifest.yaml`. We use Python standard naming convention.

Reply:
wazideploy_manifest.yaml